pkg/eval/mods/file/file.go

```diff
@@ -124,3 +127,43 @@ func prclose(p vals.Pipe) error {
 func pwclose(p vals.Pipe) error {
 	return p.WriteEnd.Close()
 }
+
+//elvdoc:fn truncate
+func truncate(name string, size vals.Num) error {
+
+	switch size.(type) {
+	case int:
+		s := int64(size.(int))
+		err := os.Truncate(name, s)
+		if err != nil {
+			return err
+		} else {
+			return nil
+		}
+
+	case *big.Int:
+		if (size.(*big.Int)).IsInt64() {
+			s := (size.(*big.Int)).Int64()
+			err := os.Truncate(name, s)
+			if err != nil {
+				return err
+			} else {
+				return nil
+			}
+		} else {
+			return errs.OutOfRange{
+				What:      "size argument for os.Truncate",
+				ValidLow:  "0",
+				ValidHigh: "8 bytes",
```

Comment:
That string doesn't make any sense. It should be derived from a constant like `MaxInt64` (see https://golang.org/pkg/math/#pkg-constants). However, the real question is whether this case should even be using `errs.OutOfRange` since it does not explicitly verify that the value is greater than or equal to zero. This is another example why the code should be structured per my previous comment so that we only execute this if the value is not a `int64` or is not negative.